cmd/policy/policy.go

```diff
@@ -501,6 +546,26 @@ This group of commands allows the management of polices to be verified against b
 	return cmd
 }
 
+func mergeCompiledConfig(globalConfig *settings.Config, processConfigOpts config.ProcessConfigOpts) ([]byte, error) {
+	var sourceConfigMap, compiledConfigMap map[string]any
+	var err error
+
+	compiler := config.New(globalConfig)
```

Comment:
I would rather this function accept a compiler instance, rather than the `settings.Config`. #dependencyInjection